using-mobile-extensions/adobe-analytics/README.md

```diff
@@ -285,21 +313,33 @@ Analytics depends on the Identity extension and is automatically included in Cor
 {% endhint %}
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
 In your app's `application:didFinishLaunchingWithOptions`, register Analytics with Mobile Core:
 
+### Swift
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+   MobileCore.registerExtensions([Analytics.self, Identity.self], {
+   MobileCore.configureWith(appId: "yourAppId") 
+ })  
+ ...
+}
+```
 ### Objective-C
 
 ```objectivec
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    [ACPCore configureWithAppId:@"yourAppId"];
-    [ACPAnalytics registerExtension];
-    [ACPIdentity registerExtension];
-    [ACPCore start:nil];
-    // Override point for customization after application launch.
-    return YES;
- }
+    [AEPMobileCore registerExtensions:@[AEPMobileAnalytics.class, AEPMobileIdentity.class] completion:^{
+    [AEPMobileCore configureWithAppId: @"yourAppId"];
+  }];
+  ...
+}
 ```
+{% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+In your app's `application:didFinishLaunchingWithOptions`, register Analytics with Mobile Core:
```

Comment:
Should the function signature be styled as:
`application(_:didFinishLaunchingWithOptions:)`
to follow the Swift conventions for function signatures? [StackOverflow discussion](https://stackoverflow.com/questions/54537403/what-is-the-meaning-of-the-in-swift-function-signatures-and-why-is-there-no-co)

Reply:
Do you mean _ application:didFinishLaunchingWithOptions?

Reply:
Updated as our discussion.